ethcore/res/ethereum/kovan.json

```diff
@@ -40,7 +40,8 @@
 		"eip140Transition": 5067000,
 		"eip211Transition": 5067000,
 		"eip214Transition": 5067000,
-		"eip658Transition": 5067000
+		"eip658Transition": 5067000,
+		"wasmActivationTransition": 6500000
```

Comment:
Should we fork on Friday? `6262000`?

Reply:
No, release will not be ready before Wed, we need at least 1 day for binaries and 1 day for tests

Reply:
Ok, let's fork on next Wed then?

Reply:
#6350000 ?

Reply:
Yeah, 6500000 is safer
we add might have some integration tests (in dedicated repos) meanwhile